### DISCUSSION TITLE
No edge tpu found

### DISCUSSION THREAD

**Discussion by x140x1n** [koush/scrypted#1476] [Q&A]
I run scrypted in an LXC on Proxmox  and the Coral M.2/PCI drivers installed, following the installation instructions.

However in the Tensorflow Lite Object Detection plugin it does not show up
![image](https://github.com/koush/scrypted/assets/7008093/18b2683b-c2f3-4494-b37d-ee1b7038d3c2)

but it is available in the LXC:

root@scrypted:~# ls -al /dev/apex_*
---------- 1 root root 0 May 16 19:17 /dev/apex_0
---------- 1 root root 0 May 16 19:17 /dev/apex_1

from the plugin console:

coral edge tpu library loaded successfully
edge tpus []
unable to use Coral Edge TPU no edge tpu found

Any idea what's causing this and how to fix it?

**Comment by koush** [comment]
Did you use the official scrypted lxc or something like tteck

**Reply by x140x1n** [reply]
the official one

**Reply by koush** [reply]
The permissions on that look incorrect. It’s root only. Did you use the proxmox coral install script in the scrypted docs?

**Comment by x140x1n** [comment]
I used the below steps as in the installation docs:

apt remove -y gasket-dkms
apt install -y git devscripts dh-dkms dkms pve-headers-$(uname -r)
cd /tmp
rm -rf gasket-driver
git clone https://github.com/google/gasket-driver.git
cd gasket-driver/
debuild -us -uc -tc -b
dpkg -i ../gasket-dkms_1.0-18_all.deb

**Reply by koush** [reply]
I think you skipped this bit during initial lxc setup https://github.com/koush/scrypted/blob/0ae93a9c3f7216fc67f74dab60add02179d58ace/install/local/install-scrypted-proxmox.sh#L71

**Reply by x140x1n** [reply]
I ran this script for the lxc setup, the udev rules file /etc/udev/rules.d/65-scrypted.rules was created

**Reply by koush** [reply]
Maybe run the last line in that script to reload and trigger.
Then restart lxc.

may wanna join discord to solve this

**Reply by x140x1n** [reply]
I have tried the reload and trigger line and restarting the lxc which made no difference.
I can't get on discord right now, but I will be able to in a few hours.
